pom.xml

```diff
@@ -469,5 +482,44 @@ file comparators, endian transformation classes, and much more.
         <coveralls.skip>true</coveralls.skip>
       </properties>
     </profile>
+    <profile>
+      <id>benchmark</id>
+      <properties>
+        <skipTests>true</skipTests>
+        <benchmark>org.apache</benchmark>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <version>1.6.0</version>
```

Comment:
Why did you pick a seemingly random old version? Why not 3.0.0, which is the current version?

Reply:
@garydgregory mybad. changed to 3.0.0 now.